Rakefile

```diff
@@ -350,17 +349,6 @@ if defined?(RuboCop::RakeTask)
   end
 end
 
-YARD::Rake::YardocTask.new(:docs) do |t|
-  # Options defined in `.yardopts` are read first, then merged with
-  # options defined here.
-  #
-  # It's recommended to define options in `.yardopts` instead of here,
-  # as `.yardopts` can be read by external YARD tools, like the
-  # hot-reload YARD server `yard server --reload`.
-
-  t.options += ['--title', "datadog #{Datadog::VERSION::STRING} documentation"]
```

Comment:
Looks like we removed this one, but didn't add it anywhere else.

This part is responsible for the title at the top of this page https://datadoghq.dev/dd-trace-rb/_index.html.

We can try moving it to the yard rakefile instead.

Reply:
It has been move to https://github.com/DataDog/dd-trace-rb/pull/3918/files#diff-738cc9f696f5a20181d325f04bc2449722c513cbdb6cda88746d6bc831c73514

Hence, calling https://github.com/DataDog/dd-trace-rb/pull/3918/files#diff-0fb7d6cd1084c2d17c22be8545371174ae11bae55b94b86033279e6bcf811060R39